ext/prism/extension.c

```diff
@@ -987,6 +987,18 @@ parse_success_p(int argc, VALUE *argv, VALUE self) {
     return result;
 }
 
+/**
+ * call-seq:
+ *   Prism::parse_failure?(source, **options) -> Array
```

Comment:
Presumably this is not correct, given that the below code only returns true or false.